examples/js/components/FileInput.jsx

```diff
@@ -4,7 +4,11 @@ import {FileInput} from 'safe-framework'
 
 class Area extends Component {
   onChange (value) {
-    window.alert(value)
+    var alertText = ' '
```

Comment:
I'll fix the 'let'. Printing the contents of the file might not be so smart because
1. binary files?
2. large files?
3. at least on the app side, the fact that I have a file object is more immediately important than being able to get at the contents.